src/current/v24.3/ldap-authorization.md

```diff
@@ -0,0 +1,113 @@
+
+## Prerequisites
+
+- Enable [LDAP Authentication]({% link v24.3/ldap-authentication.md %}).
+
+## Configuration
+
+### Step 1: Enable LDAP Authorization
+
+Add the `ldapgrouplistfilter` parameter to your HBA configuration. The configuration includes two important LDAP filters:
+
+1. `ldapsearchfilter`: Determines which users can authenticate
+2. `ldapgrouplistfilter`: Defines which groups should be considered for authorization
+
+Here's a basic example:
+
+~~~ sql
+SET CLUSTER SETTING server.host_based_authentication.configuration = '
+host    all    all    all    ldap    ldapserver=ldap.example.com 
+    ldapport=636 
+    "ldapbasedn=ou=users,dc=example,dc=com" 
+    "ldapbinddn=cn=readonly,dc=example,dc=com" 
+    ldapbindpasswd=readonly_password 
+    ldapsearchattribute=uid 
+    "ldapsearchfilter=(memberof=cn=cockroachdb_users,ou=groups,dc=example,dc=com)"
+    "ldapgrouplistfilter=(objectClass=groupOfNames)"';
+~~~
+
+For more precise control, you can configure these filters to match your security requirements:
+
+#### Search Filter Examples
+
+To restrict authentication to members of specific groups:
+
+~~~ sql
+-- Users must be members of either the database users group or the analytics team
+"ldapsearchfilter=(|(memberof=cn=cockroachdb_users,ou=groups,dc=example,dc=com)(memberof=cn=analytics_team,ou=groups,dc=example,dc=com))"
+~~~
+
+#### Group List Filter Examples
+
+The `ldapgrouplistfilter` configuration varies by LDAP server type:
+
+~~~ sql
+-- For Azure Active Directory:
+"ldapgrouplistfilter=(objectCategory=CN=Group,CN=Schema,CN=Configuration,DC=example,DC=com)"
+
+-- For OpenLDAP:
+"ldapgrouplistfilter=(objectClass=groupOfNames)"
+~~~
+
+For enhanced security, restrict the groups that can be mapped to CockroachDB roles:
+
+~~~ sql
+-- Only map specific groups to CockroachDB roles
+"ldapgrouplistfilter=(|(cn=crdb_analysts)(cn=crdb_developers))"
+~~~
+
+{{site.data.alerts.callout_info}}
+We recommend that you explicitly specify which groups should be mapped to CockroachDB roles rather than using broader filters. This ensures that only intended groups are granted database access.
+{{site.data.alerts.end}}
+
+### Step 2: Create Matching Roles
+
+Create CockroachDB roles that match your LDAP group names and grant appropriate privileges to each role. Remember that role names must comply with CockroachDB's [identifier requirements](https://www.cockroachlabs.com/docs/v24.2/create-user#user-names).
+
+For example, if you've configured the group filter to allow `crdb_analysts` and `crdb_developers`:
+
+~~~ sql
+-- Create role for analysts
+CREATE ROLE crdb_analysts;
+GRANT SELECT ON DATABASE analytics TO crdb_analysts;
+
+-- Create role for developers
+CREATE ROLE crdb_developers;
+GRANT ALL ON DATABASE app TO crdb_developers;
+~~~
+
+## Troubleshooting
+
+Enable authentication logging to troubleshoot LDAP issues:
+
+~~~ sql
+SET CLUSTER SETTING server.auth_log.sql_sessions.enabled = true;
+~~~
+
+Potential issues may pertain to:
```

Comment:
I'm not sure what we are trying to say here. Are we saying "To troubleshoot the following sorts of issues, enable logging?

Also is this "authorization logging"? Since this is the authorization page. I don't think we give the advice to enable this on the Authentication page. Should we?

Reply:
> I don't think we give the advice to enable this on the Authentication page. Should we?
We should. Right now both authn and authz log at the same place.